File: solutions/beecrowd/1096/1096.pas
Uses sysutils;

Var n, i : integer;

Begin
    for i := 1 to 9 do
    begin
        if i mod 2 <> 0 then
        begin
            writeln(Format('I=%d J=7', [i]));
            writeln(Format('I=%d J=6', [i]));
            writeln(Format('I=%d J=5', [i]));
        end;
    end;
End.

File: solutions/beecrowd/1096/1096.ml
let () =
    for i = 1 to 9 do
        if i mod 2 = 1 then (
            Printf.printf "I=%d J=7\n" i;
            Printf.printf "I=%d J=6\n" i;
            Printf.printf "I=%d J=5\n" i
        )
    done
